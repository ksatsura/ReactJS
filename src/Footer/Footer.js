//imports
import React from 'react';
import CompanyTitle from '../CompanyTitle/CompanyTitle.js';

const footer = (props) => {
    return (
        <div className='footer'>
            <CompanyTitle />
        </div>
    )
};

export default footer;