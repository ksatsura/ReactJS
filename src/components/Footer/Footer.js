//imports
import React from 'react';
import CompanyTitle from '../CompanyTitle/CompanyTitle';

const Footer = (props) => {
    return (
        <div className='footer'>
            <CompanyTitle />
        </div>
    );
};

export default Footer;