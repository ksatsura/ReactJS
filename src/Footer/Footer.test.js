import React from 'react';
import Footer from './Footer';
import {shallow} from 'enzyme';

describe('Footer.js', () => {

    describe('should render', () => {

        it('as expected', () => {
            const component = shallow(<Footer />);

            expect(component).toMatchSnapshot();
        });
    });
});