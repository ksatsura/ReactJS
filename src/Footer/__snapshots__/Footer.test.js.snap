// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.js should render as expected 1`] = `
<div
  className="footer"
>
  <Ð¡ompanyTitle />
</div>
`;
